src/main/java/com/alkemy/ong/exception/ExepcionHandler.java

```diff
@@ -61,10 +62,17 @@
         return new ResponseEntity<>(errorInfo, HttpStatus.BAD_REQUEST);
     }
     @ExceptionHandler({NumberFormatException.class})
-    public ResponseEntity<MessageInfo> notFoundExcept(HttpServletRequest request) {
+    public ResponseEntity<MessageInfo> numberFormatException(HttpServletRequest request) {
         String message = messageSource.getMessage
                 ("message.error.id.not.number", null, Locale.ENGLISH);
         MessageInfo errorInfo = new MessageInfo(message, HttpStatus.FORBIDDEN.value(), request.getRequestURI());
         return new ResponseEntity<>(errorInfo, HttpStatus.FORBIDDEN);
     }
+    @ExceptionHandler({HttpMessageNotReadableException.class})
+    public ResponseEntity<MessageInfo> messageNotReadableException(HttpServletRequest request, HttpMessageNotReadableException ex) {
+        String message = messageSource.getMessage
+                ("message.error.unexpected.character", null, Locale.ENGLISH);
+        MessageInfo errorInfo = new MessageInfo(message, HttpStatus.FORBIDDEN.value(), request.getRequestURI());
+        return new ResponseEntity<>(errorInfo, HttpStatus.FORBIDDEN);
```

Comment:
Este código de error no es correcto, porque el forbidden se debe utilizar únicamente para permisos dentro de la aplicación, es decir, si intentamos acceder a un recurso y no podemos porque no tenemos permisos para realizar dicha operación.
Lo que necesitas en este caso es un not found o un bad request, según la lógica que se implemente, si tenemos validaciones y se indica que el id ingresado es incorrecto porque está mal formado, entonces es un bad request. Si en cambio no existen estas validaciones y lo que sucede en realidad es que la base de datos no tiene ese registro y no se realiza nada más, simplemente se envía ese mensaje, en este caso es un not found.
Lo mismo con el error de arriba, el error que se captura es un NumberFormatException, es decir nunca sería un forbidden, sino un bad request

Reply:
Turco gracias, dale ahi lo modifico una disparada por BAD REQUEST.

Reply:
bad request por error en path:
![badrequestpath](https://user-images.githubusercontent.com/79908280/161465554-c9547a62-51e4-43c2-bb3c-10f1f08c31e5.png)

bad request por error en body:
![badrequestbody](https://user-images.githubusercontent.com/79908280/161465560-a122c9f9-edab-4fae-bff2-a288ef3aae32.png)